### ISSUE TITLE
Ability to clear notification targets.

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-meta#2147]
I have some pretty weird notification targets that i've racked up over the years, but apparently no way to clear them from Eleweb's UI.  It's frustrating that it knows them enough to show them to me, but not to let me delete them...

<img width="401" alt="Screenshot 2021-07-19 at 00 27 38" src="https://user-images.githubusercontent.com/1294269/126085419-24b7cc19-4aae-4527-b530-fc515c31feb7.png">

**Comment by daniellekirkwood** [comment]
@pmaier1 I think this is no longer needed as it's covered by the OIDC and other epics you've got going on? Is that the case? If yes, please go ahead and close this :)

**Comment by t3chguy** [comment]
I don't think OIDC/MAS cover notifications targets

**Comment by pmaier1** [comment]
> I don't think OIDC/MAS cover notifications targets

Correct. This is unrelated to OIDC/MAS and still handled in Synapse backend-wise.

**Comment by rollingonchrome** [comment]
I found that I could use FluffyChat (in this case on iOS) to delete any unwanted notification targets.

This is not an ideal fix. But it did tidy up my account.
